### ISSUE TITLE
Please check the received address.

### ISSUE THREAD

**Issue by arale558** [bitpay/copay#5335]
Please help me.

There is no way to tell the recipient address to the other side and the one that should have been remitted has not come into your wallet.
https://blockchain.info/address/1Bvun1oHbnsvCw6ybDv5ZiZa7CfAUjzowf

I do not know how to read the table.
What does the red arrow mean?
Does this mean that remittance is done?

My incoming address：1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy

I could receive it after remitting from my other wallet to another automatically generated address.
Likewise, if I sent it to the unknown receiving address here, I could not receive it.
Since copay's address is just copying and pasting, I think that there is no mistake. Why？。。。
I'd like to ask you what it is.

Where did it go about 2 BTC?...
Please check and lend me your wisdom.

Thank you.

Asako
from Japan

**Comment by dabura667** [comment]
日本語で状況をご説明いただけます？

**Comment by arale558** [comment]
$B$3$s$K$A$O!#(B

$BIT47$l$G!"2?$,2?$d$i%5%]!<%H$b$I$&$$$C$?;EAH$_$K$J$C$F$$$k$N$+$b$h$/J,$+$C$F$$$J$$$N$G$9$,!D(B $B$4O"Mm$$$?$@$-$"$j$,$H$&$4$6$$$^$9(B!!

$BLd$$9g$o$;$NFbMF$O!"0J2<$NDL$j$G$9!#(B

$B@hJ}$K<u?.%"%I%l%9$rEA$(!"Aw6b$5$l$F$$$k$O$:$N$b$N$,!"$*:bI[$KF~$C$FMh$F$$$^$;$s!#(B
$B@hJ}$K3NG'$7$?$H$3$m!"$3$A$i$N(Burl$B$,Aw$i$l$F$-$^$7$?!#(B
$BAw6b:Q$_$J$J$C$F$$$k$H6D$$$^$9!#(B
https://blockchain.info/address/1Bvun1oHbnsvCw6ybDv5ZiZa7CfAUjzowf

$B",(B
($B$3$A$i(Bblockchain$B$N%Z!<%8$N8+J}$b$h$/GD0.$G$-$F$$$J$$$G$9!#(B)


$B<u?.%"%I%l%9$O!"$3$A$i$G;XDj$7$F$$$^$7$?!#(B
1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy

$B9g$C$F$$$^$9!#!"(B

$B$J$*!"(Bcopay$B$G<+F0@8@.$5$l$F$$$kJL$N%"%I%l%9$K!"<+J,$NB>$N(Bwallet$B$h$jAw6b$7$?$i<u?.$G$-$F$$$^$7$?!#(B
$BF1$8$h$&$K!"$3$A$i;XDj$7$F$$$?<u?.%"%I%l%9$KAw$C$?$i<u?.$G$-$^$;$s$G$7$?!#(B

$B%"%I%l%9;XDj$,;d$NJ}$G4V0c$C$F$$$k$H$$$&$3$H$J$N$+$b$7$l$^$;$s$,!"(Bcopay$B$N%"%I%l%9$O%3%T%Z$7$F$$$k$@$1$J$N$G!"4V0c$C$F$$$J$$$H;W$&$N$G$9$,!"$I$&$$$C$?>u67$J$N$+$,Ff$G$9!#(B
$B$I$&$$$&$3$H$J$N$+860x$H$i;W$o$l$k2?$+!"$o$+$j$^$9$G$7$g$&$+!)(B

$BLs(B2BTC$B$O$I$3$K9T$C$F$7$^$C$?$N$G$7$g$&$+!)(B
$B3NG'$*4j$$$$$?$7$^$9!#(B
$BEEOC$b<u$1IU$1$^$9!#(B
$B$*<j?t$*$+$1$7$^$9$,!"2?$+J,$+$k$h$&$G$7$?$i$4;XFn$N$[$I$h$m$7$/$*4j$$$$$?$7$^$9!#(Bm(_ _)m

$B>>B<Kc;R(B
090 2308 7453


iPhone$B$+$iAw?.(B

2016/12/23 14:17$B!"(BDabura667 <notifications@github.com> $B$N%a%C%;!<%8(B:

**Comment by dabura667** [comment]
I think I was able to re-encode most of the message and I understand. She is saying someone sent her the link and told her they sent 2 BTC. Then she said something like she was sure she copy pasted directly from Copay.

The rest was all un-recoverable.

アサコ様へ

ごめんなさい、iPhoneの設定などでメール返信という形を取った返信が文字化けしているようです。
お手数ですが、返事を書く際に携帯のブラウザでもいいのでGithubにログインして実際に返事を打っていただけませんか？

文字化けを修復試みましたが、所々修復不可能な部分がありましたので、質問させて下さい：

①そのリンクを送られたと仰ったんですが、誰に送られたんですか？個人の方から送られたんですか？それとも何らかのサービスから送られたんですか？(ビットコイン販売所など)後者の場合はサイトの名前を教えていただいても宜しいでしょうか？
②Copayは取引ごとに新規のアドレスを発行するので、受取画面だと以前コピーしたアドレスが表示されない可能性が高いです。アドレスの履歴をご覧になるには、当該ウォレットのホーム画面より画面右上の歯車をタップし、「上級者向け」⇒「ウォレット詳細」で「最新ウォレットアドレス」のところまでスクロールして仰っているアドレスが見つかるか探してみて下さい。そこに無かった場合はそのウォレットのアドレスではないということになります。

送られたリンクを見る限り、2BTCが送られすぐさまに全残高を別のところに移動させたようなので、少し不安です。なので、あなたに売ろうとしていたのがどういう者かが分かれば色々と助言ができるかもしれません。

引き続き宜しくお願いします。

**Comment by arale558** [comment]
dabura667様

ご親切に感謝します。

①アドレスを送ったというか、登録しておいた先はbitregionというマッチングサイトになります。寄付をする側ともらう側のマッチングをしてくれるサイトなのですが、今回は私は受け取る側でした。
おそらく、ある方から送金されたものをbitregionを通してすぐに私に送ってくれているのかもしれません。

赤い矢印の意味するところがわからないのですが、赤矢印の先のアドレスは私のものです。

②はさっそく調べてみます。

ありがとうございます。
引き続き、よろしくお願いいたしますm(_ _)m。



****************

以下は、先に送った文字化けしてしまったものと同じ分です。

こんにちは。

不慣れで、何が何やらサポートもどういった仕組みになっているのかもよく分かっていないのですが… ご連絡いただきありがとうございます!!

問い合わせの内容は、以下の通りです。

先方に受信アドレスを伝え、送金されているはずのものが、お財布に入って来ていません。
先方に確認したところ、こちらのurlが送られてきました。
送金済みななっていると仰います。
https://blockchain.info/address/1Bvun1oHbnsvCw6ybDv5ZiZa7CfAUjzowf

↑
(こちらblockchainのページの見方もよく把握できていないです。)


受信アドレスは、こちらで指定していました。
1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy

合っています。、

なお、copayで自動生成されている別のアドレスに、自分の他のwalletより送金したら受信できていました。
同じように、こちら指定していた受信アドレスに送ったら受信できませんでした。

アドレス指定が私の方で間違っているということなのかもしれませんが、copayのアドレスはコピペしているだけなので、間違っていないと思うのですが、どういった状況なのかが謎です。
どういうことなのか原因とら思われる何か、わかりますでしょうか？

約2BTCはどこに行ってしまったのでしょうか？
確認お願いいたします。
電話も受け付けます。
お手数おかけしますが、何か分かるようでしたらご指南のほどよろしくお願いいたします。m(_ _)m

アサコ

**Comment by arale558** [comment]
dabura667様

②で教えていただいたように、確認したら、まさかまさかですがこちらのアドレスご無かったです。

他のwalletですと、blockchainしか利用していないので何故か分かりません。

ちなみに、こうしたように不明なBTCというのはどこにいってしまうのでょうか。
取り戻すのは難しそうですね。。

色々とありがとうございます。
もう少し詳しく思い返してみます。

また何かありました時には、よろしくお願いいたします。

**Comment by dabura667** [comment]
確かに 1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy にて 2.0008 BTC(現時点 207,000円程度)の残高は残っています。

お持ちのどのウォレットか。。。ですよね。。。

Copayのホーム画面から左上の三の字ボタンでウォレット切替ができますが(Copayのアプリで複数ウォレットの管理ができます)他のウォレットってありますか？あった場合、当該アドレス入っていませんか？

ちなみに、ウォレット詳細のそれぞれのアドレスの下に、xpub/0/x のようなものがあるかと思います。下に行けば行くほど低い数字になっていきます。

一番下のアドレスのxは何番ですか？
一番上は？

あと、詳細画面の一番上のウォレット ID (非常に長い16進の数字です)を教えていただければ、
Bitpay社の者にデータベースでそのアドレスが関連しているか確認してもらえるかもしれませんので、念のため教えていただけませんか？

宜しくお願いします。

**Comment by arale558** [comment]
Dabura667$BMM(B

$B$4CzG+$K$"$j$,$H$&$4$6$$$^$9!#(B

copay$B$N%&%)%l%C%H$O$R$H$D$7$+:n@.$7$F$$$J$$$G$9!#(B
copay$B$G$O$J$$$N$+$b$7$l$J$$!)!*$G$9$,;W$$=P$;$J$$$G$9!#!#(B
$B;d$N5-21$G$O!"(Bblockchain$B$N%&%)%l%C%H$rJQ$($h$&$H;W$C$F(Bcopay$B$K$7$?$O$:$G!"B>$K$O(Bbread$B$r;}$C$F$$$^$9$,MxMQ$7$?$3$H$O$^$@$"$j$^$;$s$7!"(Bcopay$B$@$C$?$H5-21$7$F$$$^$9!#(B

xpub/0/x$B$G$9$,!"#1HV2<$,(B0(10$B7n(B31$BF|IU(B)$B$G!"#1HV$&$($,(B5(12$B7n(B18$BF|IU(B)$B$K$J$j$^$9!#(B

$B$^$?!"%&%)%l%C%H(BID$B$G$9$,(B
a4722268-a9dc-4feb-a40e-0d6f1ff4df14

$B$G$9!#(B

$B$h$m$7$/$*4j$$$7$^$9(Bm(_ _)m

$B%"%5%3(B

iPhone$B$+$iAw?.(B



2016/12/25 19:24$B!"(BDabura667 <notifications@github.com> $B$N%a%C%;!<%8(B:

**Comment by dabura667** [comment]
@matiu @cmgustavo Can either of you check to see if the following address is in any way related to the following wallet ID?

Address: 1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy
Wallet ID: a4722268-a9dc-4feb-a40e-0d6f1ff4df14

@arale558 また文字化けています。。。一部修復できなかったので、引き続き返事はブラウザでお願いします。

**Comment by arale558** [comment]
Dabura667様

文字化け、、そうでしたスミマセン。

ご丁寧にありがとうございます。

copayのウォレットはひとつしか作成していないです。
copayではないのかもしれない？！ですが思い出せないです。。
私の記憶では、blockchainのウォレットを変えようと思ってcopayにしたはずで、他にはbreadを持っていますが利用したことはまだありませんし、copayだったと記憶しています。

xpub/0/xですが、１番下が0(10月31日付)で、１番うえが5(12月18日付)になります。

また、ウォレットIDですが
a4722268-a9dc-4feb-a40e-0d6f1ff4df14

です。

よろしくお願いしますm(_ _)m

アサコ

**Comment by arale558** [comment]
dabura667様

ひとつ確認ですが、こちらのサポートはどういったシステムになっているのでしょうか？

気になっていたのは、ここにアドレスIDを載せるようなことして大丈夫なのですか？

**Comment by arale558** [comment]
Hi!
Excuse me! I do not know what's going on (TT)I am grateful to you. But did the reply cease? It is! I do not know at all what this system is, I do not know what to do.



iPhoneから送信

2017/01/31 23:36、Javier Donadío <notifications@github.com> のメッセージ:

**Comment by dabura667** [comment]
@JDonadio: Sorry for the wait.

1. Someone sent this person about 2 BTC to the address 1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy
2. They say they have not yet received the money.
3. They appear to be completely new to Bitcoin (they didn't understand the basic UI of blockchain . info)
4. I asked them to check their wallet ID and which addresses were used.
5. They gave me the wallet ID a4722268-a9dc-4feb-a40e-0d6f1ff4df14
6. They said their wallet shows xpub/0/5 as the last address. generated on 12/18/2016

Also something I find fishy:

If 6th address was generated on 12/18 and the second transaction to the 1G7 address was also on 12/18... this leads me to believe that maybe the user thinks they are copying their wallet address and are being replaced by malware... because it makes no sense for 12/18 for them to generate a new address but also receive to an address generated on or before 12/09... Also, if it was malware you would think the thief would move them by now.

So if you could check the mongoDB for the wallet ID and see if the address I wrote above is contained in the wallet? If it is not, then I am guessing they got malware.

@arale558 お待たせして申し訳ございません。

wallet ID 及び アドレスがバレることで唯一損なわれるのがプライバシーです。
arale558というgithubアカウントの人が２ BTCを持っているという事実くらいしかわかりません。

しかし、Copayの裏のサーバーを運用しているBitpay社にwallet ID を教えることで、データベースの中のあなたのウォレットを見て、全てのアドレスを見て、当該アドレスが含まれているかなどの情報を調べられます。

Copayのサポートはこちらで合っていますが、公共の掲示板なので、個人情報等は避けた方がよろしいかと思います。

もし差し支え無ければ、匿名で通話しながらお手伝いもできます。
もし、この通知を見てから１時間以内に来ていただければ https://appear.in/copay-help にてお待ちしています。 (デフォルトの設定だと音声と映像が両方共有されますが、入る際に映像消したり音声消したりできます。これから１時間ほどそちらで待機しておりますので、入室したら声をかけて下さい。)

**Comment by JDonadio** [comment]
Thanks for the update, we'll check it ASAP. In the meantime did he tried cleaning the history and scanning the addresses?

**Comment by dabura667** [comment]
@arale558 
①上級者向け⇒取引履歴⇒キャッシュを消去　を１回行い
それが落ち着けば(グルグルが終わったら)
②上級者向け⇒ウォレット詳細⇒アドレスの残高照会　を１回行ってもらってもよろしいでしょうか？

それで残高は表示されないんでしょうか？

ちなみに、xpub/0/xの部分以前見てもらいましたが、その中に1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwyのアドレスは含まれていますでしょうか？

**Comment by arale558** [comment]
@dabura667様 

ご親切にありがとうございます。

通知を見たのが遅く、通話し損ないました。せっかくのご厚意すみません。

つきましては、上記の件やってみたのですが、残高表示されず、また、こちらのアドレスも含まれていません。

自分が何かを間違えているのかもしれません。記憶乏しくもはや分からず。。はっきりとした情報お伝えできず本当に申し訳ないです。m(_ _)m

**Comment by arale558** [comment]
自分が何かを間違えているとすれば、こちらのアドレスがcopayのものではないということなるかと思いますが・・
ちなみに、1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy
のアドレスは、copayのものだったかどうかというのはそちらで分かるのでしょうか？
私の記憶ではcopay のものであったとしか考えられないのですが、もし万が一違っていたとしたらと思うと。。心苦しいです。
履歴にもないというのが不思議で仕方ありませんが、、
他に理由としてはどういったことが考えられるのでしょうか。

もうこれ以上お手間をおかけするのも申し訳ないですが(TT)
よろしくお願いいたします。

**Comment by dabura667** [comment]
例えば、Copayを一回インストールして、アドレス発行したら、携帯を初期化したりしましたか？

控えた12単語のフレーズそのものがウォレットの起点なので、今控えてある全てのフレーズでウォレットを追加してインポートしてみてはいかがでしょう？

もう一つの可能性としては、Copayでコピーしたつもりのアドレスが、相手に送るまでの間に、ウィルスやマルウェアがコピーしたアドレスではなく、ウィルス作った人のアドレスを貼り付けさせたとかも前例あります。送った時、Copayアプリに表示されたアドレスなのかを送信する前に二重目視確認しましたか？

ウィルスにしても、ずっと放置されているので、考えにくいかなと思います。大体ウィルスの作成者てすぐ回収するのに、放置されてます。

上記に書いたリンクのところに入室しますと私の携帯に通知飛びますので、お手透きの時に入室して10分程度開きっぱなしにしていただけますか？

なるはやで私も入室して対応しますので。

**Comment by arale558** [comment]
@dabura667様

すれ違ってしまいすみません！
今日明日手が空かず…日曜日のお昼までか夜遅めの時間帯にもしタイミング合うようでしたら、よろしくお願いしますm(_ _)m

ちなみに、携帯を初期化はしていないです。
また、控えていた復元フレーズでウォレットを追加してインポートしてみたのですが、"既存内のウォレットです"と出てきました。

二重目視確認はあやふやです、できていないと思います。
ウィルスの件に関してもありがとうございます。理解しました。

気になる点としては、もしかしたらアドレスをちゃんとコピーできていなかったのかも？！しれません。
が、それだとしたらxpub/0/xのところにありそうなアドレスのはずですし。
分からないですが、ウィルスのアドレスをペーストし損なった？！とか。。

もう一度確認して、また後日連絡させていただきます。

よろしくお願いいたします。

**Comment by JDonadio** [comment]
@dabura667 I can confirm that the address: `1G78tYKBhwy2Qr7JVEz17YJZMKPVvP7bwy` doesn't belong to `bws`. So probably the problem was a malware or something.

**Comment by dabura667** [comment]
@JDonadio Thanks for comfirming that. I have spoke to the user and she has given up.

@arale558
どうやら、2BTCの送付先がCopayを使って作られたアドレスでは無いそうです。データベースにそのアドレスの形跡が無いです。

ということは blockchain . info など、Copay以外のウォレットか、ウィルスやマルウェアによって盗まれたか、あのサイトの相手の方による詐欺かの三つのうちのどれかですね。

このイシューはこれ以上対応できませんのて、大変申し訳ありませんが、クローズボタンを押していただいてもよろしいでしょうか。

お手数おかけします。

**Comment by JDonadio** [comment]
Yes it's very disappointing cases like this. Thanks alot for your dedicated support work.

**Comment by arale558** [comment]
dabura667様

遅くなりました。

最後まで、親身にご対応くださり感謝いたします。本当にありがとうございました。

dabura様に良いことがあります♪
また何かありました時にはよろしくお願いいたします〜☆

**Comment by arale558** [comment]
すみません、closedされていました！ありがとうございました。m(__)m
